lib/contentful/delivery.ex

```diff
@@ -102,52 +110,57 @@ defmodule Contentful.Delivery do
   end
 
   defp process_url(url) do
-    "#{protocol}://#{endpoint}#{url}"
+    "#{@protocol}://#{@endpoint}#{url}"
   end
 
   defp process_response_body(body) do
     body
-      |> Poison.decode!
-      |> resolve_includes
+    |> Poison.decode!
+    |> resolve_includes
```

Comment:
This was not included here because when fetching assets or content types, there are no includes. Therefore this was only needed for entries.